Source/WebCore/en.lproj/Localizable.strings

```diff
@@ -1701,3 +1701,12 @@
 
 /* Lockdown Mode alert message */
 "Certain experiences and features may not function as expected. You can turn off Lockdown Mode for this app in Settings." = "Certain experiences and features may not function as expected. You can turn off Lockdown Mode for this app in Settings.";
+"Single Page" = "Single Page";
```

Comment:
Nit: This should be moved under your `PDFJS strings` comment.

Reply:
Moved

Reply:
`/* PDFJS strings*/` -> `/* PDFJS strings */